cratedb_toolkit/api/main.py

```diff
@@ -129,7 +129,11 @@ def load_table(
                 logger.error("Data loading failed or incomplete")
                 return False
 
-        elif source_url_obj.scheme.startswith("influxdb"):
+        elif (
+            source_url_obj.scheme.startswith("influxdb")
+            or resource.url.endswith(".lp")
+            or resource.url.endswith(".lp.gz")
+        ):
```

Comment:
The feature has been there already, and only needed to be unlocked here by adjusting the dispatching.